OrbitGl/GlPanel.cpp

```diff
@@ -6,23 +6,20 @@
 
 #include "CaptureWindow.h"
 
-GlPanel* GlPanel::Create(Type a_Type) {
-  GlPanel* panel = nullptr;
+std::unique_ptr<GlPanel> GlPanel::Create(Type a_Type) {
+  std::unique_ptr<GlPanel> panel = nullptr;
```

Comment:
if (a_Type != CAPTURE) return nullptr;

and then you can just assume this is capture - construct and return it - switch is no longer needed.

Reply:
I think the "switch" made it more clear on what the intention of this method initially was if more types of panels would be added. But when looking at your comment, I realized that this will crash badly if anything else but CAPTURE would be passed here in the future, so I changed it as recommended and added a comment to clarify the intent of this.